crux-test/test/logback-test.xml

```diff
@@ -1,22 +1,46 @@
 <configuration>
-  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-    <encoder>
-      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
-    </encoder>
-  </appender>
-  <root level="WARN">
-    <appender-ref ref="STDOUT" />
-  </root>
-  <logger name="crux" level="INFO" />
-  <logger name="crux.node" level="WARN" />
-  <logger name="crux.standalone" level="ERROR" />
-  <logger name="crux.hash" level="WARN" />
-  <logger name="crux.http-server" level="WARN" />
-  <logger name="crux.kafka" level="WARN" />
-  <logger name="crux.jdbc" level="WARN" />
-  <logger name="crux.kv.memdb" level="ERROR" />
-  <logger name="crux.rdf" level="ERROR" />
-  <logger name="crux.tx" level="WARN" />
-  <logger name="org.apache.kafka" level="ERROR" />
-  <logger name="org.apache.zookeeper" level="ERROR" />
+    <appender name="FILE" class="ch.qos.logback.core.FileAppender">
```

Comment:
rather than changing the log levels of all the tests, it might be better here to wrap the test code in `wrap-redefs` to override the behaviour of the log function. if memory serves, all of the various `log/*` calls eventually call a lower-level function (`log/log`, or something like that) - you can then store whatever's logged into an atom and search through it from there.

Reply:
hey @t-taylor - this is still going to output log files for all the other tests that don't have a `remove-log-file` fixture